.travis.yml

```diff
@@ -12,12 +12,16 @@ env:
   - PLATFORMIO_ENV=esp12e
   - PLATFORMIO_ENV=nodemcuv2
 
+before_install:
+  - nvm install 6
```

Comment:
This is also fine in install.

Reply:
@janLo any comment here? please change

Reply:
Its taken from the TravisCI doc: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Dependency-Management

Reply:
I don't see any reason, why it can not go into `install` but I can test it...